src/Features/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -152,7 +153,8 @@ internal abstract partial class AbstractPullDiagnosticHandler<TDiagnosticsParams
 
             // Next process each file in priority order. Determine if diagnostics are changed or unchanged since the
             // last time we notified the client.  Report back either to the client so they can update accordingly.
-            var orderedSources = await GetOrderedDiagnosticSourcesAsync(context, cancellationToken).ConfigureAwait(false);
+            var orderedSources = await GetOrderedDiagnosticSourcesAsync(
```

Comment:
can we log the diagnostic kind somewhere in here?

Reply:
done.